runtime/tests/tools/ggma_run/src/args.cc

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "args.h"
+#include "nnfw.h"
+
+#include <iostream>
+#include <string>
+#include <sys/stat.h>
+
+#define NNPR_ENSURE_STATUS(a)        \
+  do                                 \
+  {                                  \
+    if ((a) != NNFW_STATUS_NO_ERROR) \
+    {                                \
+      exit(-1);                      \
+    }                                \
+  } while (0)
+
```

Comment:
Beyond this PR.

The `NNPR_ENSURE_STATUS` definition appears in several files:
- runtime/tests/tools/ggma_run/src/ggma_run.cc
- runtime/tests/tools/onert_run/src/nnfw_util.h
- runtime/tests/tools/onert_train/src/nnfw_util.h
- runtime/tests/custom_op/FillFrom/FillFrom_runner.cc
- runtime/contrib/style_transfer_app/src/style_transfer_app.cc

This is why I tried to define `GGMA_ENSURE` in `ggma_macro.h`. To minimize the expose API, however, we could follow the suggestion in #16260 and place it under `test/tools/libs` instead.